gameoflife.php

```diff
@@ -54,24 +69,19 @@
     if ($height < 10) $height = 10;
 }
 
-if ($options->getOption("maxSteps"))
+if ($options->getOption("maxSteps") != null)
 {
     $maxSteps = intval($options->getOption("maxSteps"));
 }
 
-if ($options->getOption("startRandom"))
+if ($options->getOption("input"))
 {
-    $startRandom = new Random("$width", "$height", "$maxSteps");
-    $startRandom->start();
-    die;
-}
+    $field = new Field($width, $height, $maxSteps);
 
-if ($options->getOption("startGlider"))
-{
-    $startGlider = new Glider("$width", "$height", "$maxSteps");
-    $startGlider->start();
-    die;
-}
+    if (array_key_exists($options->getOption("input"), $allInputs))
```

Comment:
That is perfectly valid code, but you could also have used the shorter `isset($allInputs[$options->getOption("input")])` syntax.